Source/WarpX.H

```diff
@@ -265,6 +274,12 @@ public:
 
     static amrex::Real quantum_xi_c2;
 
+
+    // Load balance options
+    enum LBType { UNDEFINED = -1, DEFAULT, HEURISTIC };
```

Comment:
Can you please add a doxygen string to this enums?
[Here is a small example.](https://github.com/openPMD/openPMD-api/blob/a545854367624984555a1d2a06463e88796df3fd/include/openPMD/IO/AccessType.hpp)

You might also want to make this an `enum struct` or `enum class` in modern C++ :)

Reply:
Documentation added.  Converted to `enum class` to be modern =)